packages/vuetify/src/composables/group.ts

```diff
@@ -0,0 +1,197 @@
+// Utilities
+import { reactive, provide, inject, computed, onBeforeUnmount } from 'vue'
+import { wrapInArray, uuid, deepEqual } from '@/util/helpers'
+import { consoleWarn } from '@/util/console'
+import { useProxiedModel } from './proxiedModel'
+
+// Types
+import type { Ref, UnwrapRef, InjectionKey, SetupContext } from 'vue'
+
+interface GroupItem {
+  id: string
+  value: Ref<unknown>
+}
+
+interface GroupProps {
+  modelValue?: unknown
+  multiple?: boolean
+  mandatory?: boolean
+  max?: number
+}
+
+interface GroupProvide {
+  register: (item: GroupItem) => void
+  unregister: (id: string) => void
+  toggle: (id: string) => void
+  selected: Ref<any[]>
+  isSelected: (id: string) => boolean
+  prev: () => void
+  next: () => void
+}
+
+export function useGroupItem (
+  props: { value?: unknown },
+  injectKey: InjectionKey<GroupProvide>,
+) {
+  const group = inject(injectKey)
+
+  if (!group) {
+    throw new Error(`Could not find useGroup injection for symbol ${injectKey}`)
+  }
+
+  const id = uuid()
```

Comment:
Is this supposed to replace `vm._uid`? It seems like overkill to me and probably causes worse problems in tests if it ends up in the DOM because now it's random. Something like

```js
export function getUid () {
  return getUid._uid++
}
getUid._uid = 0
```

would be fine and then we can call `beforeEach(() => getUid._uid = 0)` to reset it before each test.